pkg/operator/ceph/version/version.go

```diff
@@ -150,3 +166,58 @@ func (v *CephVersion) IsAtLeastNautilus() bool {
 func (v *CephVersion) IsAtLeastMimic() bool {
 	return v.IsAtLeast(Mimic)
 }
+
+// IsIdentical checks if Ceph versions are identical
+func IsIdentical(a, b CephVersion) bool {
+	if a.Major == b.Major {
+		if a.Minor == b.Minor {
+			if a.Extra == b.Extra {
+				return true
+			}
+		}
+	}
+
+	return false
+}
+
+// IsSuperior checks if a given version if superior to another one
+func IsSuperior(a, b CephVersion) bool {
+	if a.Major > b.Major {
+		return true
+	}
+	if a.Major == b.Major {
+		if a.Minor > b.Minor {
+			return true
+		}
+	}
+	if a.Major == b.Major {
+		if a.Minor == b.Minor {
+			if a.Extra > b.Extra {
+				return true
+			}
+		}
+	}
+
+	return false
+}
+
+// IsInferior checks if a given version if inferior to another one
+func IsInferior(a, b CephVersion) bool {
+	if a.Major < b.Major {
+		return true
+	}
+	if a.Major == b.Major {
+		if a.Minor < b.Minor {
+			return true
+		}
+	}
+	if a.Major == b.Major {
+		if a.Minor == b.Minor {
```

Comment:
nit: you could delete lines 213 and 214 and include the minor check in the previous `if` block for equal major version.